Makefile

```diff
@@ -43,16 +46,17 @@ help:
 	@echo "  doctest    to run all doctests embedded in the documentation (if enabled)"
 	@echo "  coverage   to run coverage check of the documentation (if enabled)"
 	@echo "  dummy      to check syntax errors of document sources"
+	@echo "  images     to make drawio images"
 
 .PHONY: clean
 clean:
 	rm -rf $(BUILDDIR)/*
 
 .PHONY: html
 html:
-	$(SPHINXBUILD) -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
+	$(SPHINXBUILD) -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html && ./$(DRAWIOBUILDPATH)
```

Comment:
This should be updated to instead just use bash directly instead of python.

```make
html:
         $(SPHINXBUILD) -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
         find $(BUILDDIR)/html -name "*.drawio" -print | xargs -I {} draw.io -x -f "png" {}
````